engine/src/main/resources/assets/shaders/highp_frag.glsl

```diff
@@ -28,10 +28,13 @@ void main() {
     //    discard;
     //}
 
-    vec3 brightColor = max(color.rgb - vec3(highPassThreshold), vec3(0.0));
-    float bright = dot(brightColor, vec3(1.0));
+   // vec3 brightColor = max(color.rgb - vec3(highPassThreshold), vec3(0.0));
+    float bright = dot(color.rgb, vec3(0.2126, 0.7152, 0.0722));;
```

Comment:
This is relative luminance (not brightness) and is approximated for RGB colors using this simple formula. Green is assumed to be perceived as the brightest of the three, so it counts the most, blue the darkest.
https://en.wikipedia.org/wiki/Relative_luminance